andromedocs-theme/public/page-data/index/page-data.json

```
@@ -0,0 +1 @@
+{"componentChunkName":"component---src-pages-index-mdx","path":"/","result":{"pageContext":{"frontmatter":{"title":"Hello world!","path":"/page","date":"2020-10-01T00:00:00.000Z","author":"Vtex"}}},"staticQueryHashes":[]}
```

```suggestion
{"componentChunkName":"component---src-pages-index-mdx","path":"/","result":{"pageContext":{"frontmatter":{"title":"Hello world!","path":"/page","date":"2020-10-01T00:00:00.000Z","author":"VTEX"}}},"staticQueryHashes":[]}
```

could you also format the json please? :pleading_face:

---

resolved 4b90437